mysql-test/suite/innodb/t/log_corruption_1.test

```diff
@@ -30,7 +30,7 @@ EOF
 # platform sporadically
```

Comment:
What is the path of this file in 5.7 HEAD?

Reply:
This is a new 8.0 file but the problem is the same `sync_latch_meta_init()` memory leak.